### PR TITLE
Finished code challenge 11

### PR DESCRIPTION
**Pull request by PotatoGod123**
- [x ] Top-level README “Table of Contents” is updated
 - [x ] Feature tasks for this challenge are completed
 - [x ] Unit tests written and passing
     - [ x] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ x] Edge Case (if applicable/obvious)
 - [x ] README for this challenge is complete
     - [x ] Summary, Description, Approach & Efficiency, Solution
     - [x ] Link to code
     - [x ] Picture of whiteboard

### CODE REVIEW

File: challenges/test/lib/src/test/java/code/stacksandqueuestest/PseudoQueueTest.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package code.stacksandqueuestest;
+
+import code.stacksandqueues.PseudoQueue;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class PseudoQueueTest {
```
Comment (Mattpet26) [inline challenges/test/lib/src/test/java/code/stacksandqueuestest/PseudoQueueTest.java:7]: You wrote out an exception on the dequeue method, don't forget to test it! What happens if I try to dequeue when everything is empty? Does the exception work or do we get an error?
